client/python/gradio_client/client.py

```diff
@@ -59,17 +59,20 @@ def __init__(
         src: str,
         hf_token: str | None = None,
         max_workers: int = 40,
+        serialize: bool = True,
         verbose: bool = True,
     ):
         """
         Parameters:
             src: Either the name of the Hugging Face Space to load, (e.g. "abidlabs/whisper-large-v2") or the full URL (including "http" or "https") of the hosted Gradio app to load (e.g. "http://mydomain.com/app" or "https://bec81a83-5b5c-471e.gradio.live/").
             hf_token: The Hugging Face token to use to access private Spaces. Automatically fetched if you are logged in via the Hugging Face Hub CLI. Obtain from: https://huggingface.co/settings/token
             max_workers: The maximum number of thread workers that can be used to make requests to the remote Gradio app simultaneously.
+            serialize: Whether the client should serialize the inputs and deserialize the outputs of the remote API. If set to False, the client will pass the inputs and outputs as-is, without serializing/deserializing them. E.g. you if you set this to False, you'd submit an image in base64 format instead of a filepath, and you'd get back an image in base64 format from the remote API instead of a filepath.
```

Comment:
Tested with images and `serialize=False` is working

![image](https://user-images.githubusercontent.com/41651716/231896983-efa6e6fc-93d7-48c8-b1c3-2e3d20c1f621.png)